core/tests/puppeteer-acceptance-tests/spec/exploration-creator-tests/create-exploration-and-change-basic-settings.spec.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for Exploration Creator and Exploration Manager
+ */
+
+import testConstants from '../../puppeteer-testing-utilities/test-constants';
+import {UserFactory} from '../../puppeteer-testing-utilities/user-factory';
+import {ExplorationCreator} from '../../user-utilities/exploration-creator-utils';
+import {SuperAdmin} from '../../user-utilities/super-admin-utils';
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+
+describe('Exploration Creator', function () {
+  let explorationCreator: ExplorationCreator;
+  let explorationVisitor: ExplorationCreator;
+  let superAdmin: SuperAdmin;
+
+  beforeAll(async function () {
+    explorationCreator = await UserFactory.createNewUser(
+      'explorationAdm',
+      'exploration_creator@example.com'
+    );
+    explorationVisitor = await UserFactory.createNewUser(
+      'explorationSeeker',
+      'exploration_visitor@example.com'
+    );
+    superAdmin = await UserFactory.createNewSuperAdmin('Leader');
+    const guestUser1 = await UserFactory.createNewUser(
+      'guestUsr1',
+      'guest_user1@example.com'
+    );
+    await guestUser1.closeBrowser();
+
+    const guestUser2 = await UserFactory.createNewUser(
+      'guestUsr2',
+      'guest_user2@example.com'
+    );
+    await guestUser2.closeBrowser();
+
+    const guestUser3 = await UserFactory.createNewUser(
+      'guestUsr3',
+      'guest_user3@example.com'
+    );
+    await guestUser3.closeBrowser();
+
+    await superAdmin.assignRoleToUser('explorationAdm', 'voiceover admin');
```

Comment:
Why does the same user need to be both curriculum and voiceover admin? Can we have a curriculum admin user and a voiceover admin user? (This is more commonly what happens in practice.)

Reply:
corrected. As We can add voiceover artist with only voiceover admin role. Thanks :)

Reply:
The exploration creator needs to assign a voiceover artist, so he requires the 'voiceover admin' role. Additionally, after the exploration is published, if the creator needs to delete it, he also needs the 'curriculum admin' role.

Reply:
I think you should be able to create separate users that do each of the different tasks, right?